src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.card__rating-stars {
+  display: flex;
+}
+
+.card__rating-star {
+  background-image: url(/src/images/star.svg);
+  width: $star-size;
+  height: $star-size;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+
+  &:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+  }
+}
```

Comment:
You should have `stars` block styles here
Not styles that are related to `card` block